scripts/install-worker.sh

```diff
@@ -328,6 +359,64 @@ if [[ ! $KUBERNETES_VERSION =~ "1.19"* && ! $KUBERNETES_VERSION =~ "1.20"* && !
     sudo mv $TEMPLATE_DIR/ecr-credential-provider-config /etc/eks/ecr-credential-provider/ecr-credential-provider-config
 fi
 
+################################################################################
+### Cache Images ###############################################################
+################################################################################
+if [[ "$CACHE_CONTAINER_IMAGES" = "true" && "$BINARY_BUCKET_REGION" != "us-iso-east-1" && "$BINARY_BUCKET_REGION" != "us-isob-east-1" ]]; then
```

Comment:
This results in the following images cached https://gist.github.com/bwagner5/e6415ec62e9eed3c864f39077346ce9f

Reply:
nit, most of the string compare with syntax `[[ "$a" == "xxx" ]]` but here we use `[[ "$a" = "true" ]]`.